tests/lib/prepare-project.sh

```diff
@@ -66,6 +66,42 @@ fedora_build_rpm() {
     rm $GOPATH/*.src.rpm
 }
 
+opensuse_build_rpm() {
+    release=$(echo "$SPREAD_SYSTEM" | awk '{split($0,a,"-");print a[2]}')
```

Comment:
It's a reliable identifier we own. /etc/os-release isn't really owned by us. Would like to keep these influence out of the picture when we're doing spread specific things like setting up the environment.